staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```
@@ -347,7 +356,9 @@ func getAllDefaultRoutes() ([]Route, error) {
 	v6Routes, _ := v6File.extract()
 	routes = append(routes, v6Routes...)
 	if len(routes) == 0 {
-		return nil, fmt.Errorf("No default routes.")
+		return nil, &NoRoutesError{
```

usually errors are not returned as pointers.